checkout-v3/index.md

```diff
@@ -126,8 +136,6 @@ below to read more." %}
 [swp-logo]: /assets/img/swedbank-pay-vertical-black.svg
 [swish-logo]: /assets/img/icon-swish-simple.svg
 [trustly-logo]: /assets/img/icon-trustly-simple.svg
-[wcag-presentation]: https://www.swedbankpay.com/information/wcag
-[paid]: /checkout-v3/features/technical-reference/resource-sub-models#paid
```

Comment:
Autoremoved, seems these links are not in use and were therefore removed automatically